### PR TITLE
fix html templates

### PR DESCRIPTION
**Pull request by matmair**
Makes parsing for screenreads easier and has better w3c compliance.

### CODE REVIEW

File: InvenTree/part/templates/part/part_pricing.html [hunk 1]
```diff
@@ -9,11 +9,11 @@
 
 <table class='table table-striped table-condensed table-price-two'>
     <tr>
-        <td><b>{% trans 'Part' %}</b></td>
+        <td><strong>{% trans 'Part' %}</b></td>
```
Comment (SchrodingersGat) [inline InvenTree/part/templates/part/part_pricing.html:12]: Should the `<strong>` be closed with `</strong>` and not `</b>`?

Comment (matmair) [reply InvenTree/part/templates/part/part_pricing.html]: @SchrodingersGat you absolutley right, seems like I forgot to push that

**Comment by SchrodingersGat** [comment]
Thanks. Are you planning on making these changes for more files, or just this one?

**Comment by matmair** [comment]
@SchrodingersGat generally I would like to fix it everywhere but would wait till we have the new merging strategy and are stable. So in 0.6.0 or so.

I started using a source-analyzer (if you are interested you can find it [here](https://sonarcloud.io/dashboard?id=matmair_InvenTree)) I already use at work for my commits to InvenTree recently and it complained about those tags.

**Pull request merged**